x-pack/plugins/ml/public/application/components/annotations/annotations_table/annotations_table.js

```diff
@@ -126,6 +143,56 @@ export class AnnotationsTable extends Component {
     return mlJobService.getJob(jobId);
   }
 
+  handleSearchChange({ query, queryText, error }) {
+    if (error) {
+      this.setState({ searchError: error });
```

Comment:
I think return true should still work since if `EuiInMemoryTable.error` is undefined, the table will show the error message automatically. But I'll add a comment to note why.

<img width="1140" alt="Screen Shot 2020-07-01 at 1 30 36 PM" src="https://user-images.githubusercontent.com/43350163/86279265-5ba5ad00-bb9f-11ea-8943-911072edb1be.png">